os/include/tinyara/pm/pm.h

```diff
@@ -635,6 +635,23 @@ enum pm_state_e pm_querystate(int domain);
  ****************************************************************************/
 void pm_dvfs(int div_lvl);
 
+/****************************************************************************
+ * Name: up_set_dvfs
+ *
+ * Description:
+ *   BSP operation called from wrapper API pm_dvfs() to reduce the clock 
+ *   frequency of the AP core (CA32) of AI_Dual chipset.
+ *
+ * Input Parameters:
+ *   div_lvl - voltage frequency scaling level
+ *
+ * Returned Value:
+ *   None.
+ *
+ *
+ ****************************************************************************/
+void up_set_dvfs(int div_lvl);
+
```

Comment:
Please take a look at the function description for pm_dvfs() in pm_dvfs.c and the header file and make any changes required to make the description accurate.